From cd9673df261fca38b3cac5d9c2f28ce830cb9998 Mon Sep 17 00:00:00 2001
From: Tubbles <jae91m@gmail.com>
Date: Sat, 19 Mar 2022 16:55:28 +0100
Subject: [PATCH] Also clear clipboard along with selection

---
 x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/x.c b/x.c
index 23c2508..42447c4 100644
--- a/x.c
+++ b/x.c
@@ -687,6 +687,7 @@ setsel(char *str, Time t)
 	XSetSelectionOwner(xw.dpy, XA_PRIMARY, xw.win, t);
 	if (XGetSelectionOwner(xw.dpy, XA_PRIMARY) != xw.win)
 		selclear();
+	clipcopy(NULL);
 }
 
 void
-- 
2.35.1

