From 9162097bc4fccd6163314d13e2866c93b56ef697 Mon Sep 17 00:00:00 2001
From: Tubbles <jae91m@gmail.com>
Date: Sat, 19 Mar 2022 16:56:39 +0100
Subject: [PATCH] Change font to DejaVuSansMono

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 91ab8ca..78103c3 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "DejaVuSansMono:pixelsize=13:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
-- 
2.35.1

