From e8d4b860c6029a7c92fa8600cc8ba1312bbe9e9e Mon Sep 17 00:00:00 2001
From: Tubbles <jae91m@gmail.com>
Date: Sat, 19 Mar 2022 18:25:47 +0100
Subject: [PATCH] Add spoiler support

---
 x.c | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/x.c b/x.c
index 48025f1..d8a260c 100644
--- a/x.c
+++ b/x.c
@@ -1554,9 +1554,14 @@ xdrawglyphfontspecs(const XftGlyphFontSpec *specs, Glyph base, int len, int x, i
 	}
 
 	if (base.mode & ATTR_REVERSE) {
-		temp = fg;
-		fg = bg;
-		bg = temp;
+		if (bg == fg) {
+			bg = &dc.col[defaultfg];
+			fg = &dc.col[defaultbg];
+		} else {
+			temp = fg;
+			fg = bg;
+			bg = temp;
+		}
 	}
 
 	if (base.mode & ATTR_BLINK && win.mode & MODE_BLINK)
-- 
2.35.1

